index.html

```diff
@@ -292,5 +292,46 @@ <h4 class="titleSM">LET’S CONNECT</h4>
         </ul>
       </div>
     </section>
+    <section class="contactForm">
+      <div class="contactBg">
+        <div class="contactTop">
+          <h3 class="contactTitle">Contact me</h3>
```

Comment:
- [ ] Great job integrating the form with Formspree service. :+1: Let's work on minor details to match the design. Kindly increase the space above this title `Contact me`. Please also adjust the space between the paragraph and the text input. ✍️

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/17243325/220429757-ac290f24-e959-4fc9-bde9-ff02bd6c59dd.png" width="300">|<img src="https://user-images.githubusercontent.com/17243325/220429765-8f0ff64f-9e0f-42f5-bd15-27b3a887394b.png" width="300">|

Reply:
Hi @amejid 

Thank you for the detailed review. I shall make these changes and make a new request.